sympy/sets/sets.py

```diff
@@ -92,6 +92,12 @@ def intersect(self, other):
         >>> Interval(1, 3).intersect(Interval(1, 2))
         [1, 2]
 
+        >>> from sympy import imageset, Lambda
+
```

Comment:
Please remove this unnecessary blank line. This results in a code-block split in docs.